playbook/app/pb_kits/playbook/pb_dialog/dialog.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="pb_dialog_wrapper_rails">
+    <%= content_tag(:dialog,
+        aria: object.aria,
+        data: object.data,
+        id: object.id,
+        class: object.classname) do %>
+        <%= pb_rails("flex", props: {classname:"dialog_header", spacing:"between", padding:"sm"}) do %>
+                <%= object.title %>
+                <form method="dialog"><button class="dialog-button-class"><%= pb_rails("icon", props:{icon: "times"}) %></button></form>
```

Comment:
Let's hop on a zoom to talk through how to make this more flexible.
I think we will need to follow a structure similar to our other compound components like the Card, etc.

Reply:
I separated it out into partials like you asked! I also added the complex example because I wanted to test whether or not the dialog body would accept other kits since that's probably a more needed feature for the dialog.